test/unit/cardinality_test.rb

```diff
@@ -1,23 +1,35 @@
 require File.expand_path('../../test_helper', __FILE__)
 require 'mocha/cardinality'
+require 'mocha/return_values'
+require 'mocha/yield_parameters'
```

Comment:
Minor: There should be a `require 'mocha/invocation'` statement here. It looks like you've added it in a later commit. I'll fix this up.

Reply:
Fixed up in 836f85b8514c841781767c1d31e923af5074d4ae.